libcontainer/cgroups/utils.go

```diff
@@ -601,5 +604,22 @@ func ConvertBlkIOToCgroupV2Value(blkIoWeight uint16) uint64 {
 // convert from [2-262144] to [1-10000]
 // 262144 comes from Linux kernel definition "#define MAX_SHARES (1UL << 18)"
 func ConvertCPUSharesToCgroupV2Value(cpuShares uint64) uint64 {
+	if cpuShares == 0 {
+		return 0
+	}
 	return (1 + ((cpuShares-2)*9999)/262142)
 }
+
+// ConvertCPUQuotaCPUPeriodToCgroupV2Value generates cpu.max string.
+func ConvertCPUQuotaCPUPeriodToCgroupV2Value(quota int64, period uint64) string {
+	if quota <= 0 && period == 0 {
+		return ""
+	}
+	if period == 0 {
+		period = 100000
```

Comment:
(just a comment as I was glancing over the changes); would it be a good idea to define `const`'s for these magic values? (there's some other values above as well)

Reply:
I think just adding comments is fine. Not really reusable as `const` values.

Reply:
added code comment for explanation